Basic/2DSpaceShooter/README.md

```diff
@@ -1,14 +1,31 @@
-## 2D Space Shooter
+# 2D Space Shooter
 
-This is a UNet sample project converted to Netcode for GameObjects.
+![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)
+![NetcodeVersion](https://img.shields.io/badge/Netcode%20Version:-1.2.0-57b9d3.svg?logo=unity&color=2196F3)
+[![LatestRelease](https://img.shields.io/badge/Latest%20%20Github%20Release:-v1.2.0-57b9d3.svg?logo=github&color=brightgreen)](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/releases/tag/v1.2.0)
+<br><br>
 
-## Sample Features
+This is a UNet sample project converted to Netcode for GameObjects. The 2DSpaceShooter sample is a bitesize sample designed to demonstrate networked 2D and physics-based character movement. 
+<br><br>
 
-This sample shows you:
+# Sample Overview
```

Comment:
Yay for standardisation of these sections!🥇

Reply:
Could we also add a little header image for each sample?

Reply:
We might not have time to make new header images, but maybe we can reuse the little square images that we use for the readmes?

Reply:
there's not time to add header images for each of these, maybe with the next release!